### DISCUSSION TITLE
VS Code auto-imports from .prisma/client instead of @prisma/client

### DISCUSSION THREAD

**Discussion by jacobbrunson** [prisma/prisma#10311] [Q&A]
I added Prisma to my TypeScript project. I noticed it generates a `.prisma` modules folder in addition to the `@prisma` modules folder.

Whenever I try to auto-import any of the generated model types using VS Code IntelliSense, it always imports from `.prisma/client` rather than the preferred `@prisma/client`

Anyone else run into this problem? I thought there might be some VS Code setting or a tsconfig option to work around this, but I haven’t found any solution.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
